### PR TITLE
Add imgpatchtools-git

### PR DESCRIPTION
**Pull request by simonsmh**
AUR: https://aur.archlinux.org/packages/imgpatchtools-git

### CODE REVIEW

**Comment by yan12125** [comment]
This package should be called imgpatchtools-git.

File: archlinuxcn/imgpatchtools/PKGBUILD [hunk 1]
```diff
@@ -19,7 +19,14 @@ pkgver() {
 
 build() {
 	cd "$pkgname"
-	make all
+	mkdir -p bin
+	for dir in "applypatch" "android-base" "edify" "minzip" "otafault" "blockimg"
+	do
+		pushd $dir
+		make -j
```
Comment (lilydjwg) [inline archlinuxcn/imgpatchtools/PKGBUILD:26]: Why `-j`???

**Comment by simonsmh** [comment]
如果改了名，那现有的这个AUR包应该怎么办？

**Comment by imlonghao** [comment]
开一个新的 imgpatchtools-git 然后申请把 imgpatchtools Delete 或者 Merge 去带 -git 的

**Comment by yan12125** [comment]
Thanks!

**Pull request merged**